apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceHeaderPropagatingHandlerBase.cs

```diff
@@ -25,72 +25,52 @@ namespace Google.Cloud.Diagnostics.Common
     /// Traces outgoing HTTP requests and propagates the trace header.
     /// </summary>
     ///
-    /// <example>
-    /// <code>
-    /// public void DoSomething(IManagedTracer tracer)
-    /// {
-    ///     var traceHeaderHandler = TraceHeaderPropagatingHandler.Create(tracer);
-    ///     using (var httpClient = new HttpClient(traceHeaderHandler))
-    ///     {
-    ///         ...
-    ///     }
-    /// }
-    /// </code>
-    /// </example>
     /// <remarks>
     /// Ensures the trace header is propagated in the headers for outgoing HTTP requests and 
     /// traces the total time of the outgoing HTTP request.  This is only done if tracing is initialized
     /// and tracing is enabled for the request current request.
     /// </remarks>
-    public class TraceHeaderPropagatingHandler : DelegatingHandler
+    public class TraceHeaderPropagatingHandlerBase : DelegatingHandler
```

Comment:
I removed the base class and had ASP.NET Core create an instance for dependency injection and added a static function to CloudTrace to create the handler for  ASP.NET.